rtos/TARGET_CORTEX/rtx5/RTX/Source/TOOLCHAIN_GCC/TARGET_CORTEX_A/irq_ca.S

```diff
@@ -391,13 +391,13 @@ osRtxContextSave:
 
                 VMRS    R2, FPSCR
                 STMDB   R3!, {R2,R12}              // Push FPSCR, maintain 8-byte alignment
-                #if TARGET_FEATURE_EXTENSION_REGISTER_COUNT == 16
+                #if TARGET_FEATURE_EXTENSION_REGISTER_COUNT_GCC == 16
```

Comment:
We fixed that issue in the meanwhile. Please check with latest CMSIS release if this works for you.